### PR TITLE
Uncucks jons upstream (by codecucking him)

### PR DESCRIPTION
**Pull request by GuyonBroadway**
Everything works so far, but I will post daemon output

~~Also I can't remove items from butts just yet so I'm still trying to figure that out~~
I was able to get this sorted

A few PRs that make many many changes:
PR for the storage component: https://github.com/tgstation/tgstation/pull/36696
PR for /obj/item/device refactor: https://github.com/tgstation/tgstation/pull/37297
PR for uppercase slots: https://github.com/tgstation/tgstation/pull/37476
PR for renaming traits to quirks: https://github.com/tgstation/tgstation/pull/37646
PR for removal of stock market: https://github.com/tgstation/tgstation/pull/37414

compiling:
image

server output:
image

Output so far:
[server_log.txt](https://github.com/HippieStation/HippieStation/files/2019640/server_log.txt)

game:
image

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Oh shit you fixed it?

**Comment by GuyonBroadway** [comment]
Nope
it just undid the conflicts, i'm now looking as to WHY build_dependancies fails

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,71 @@
 # This list auto requests reviews from the specified org members
 # when a PR that modifies the file in question is opened
-# This list is alphabetized by User -> Filename KEEP IT THAT WAY
\ No newline at end of file
+# This list is alphabetized by User -> Filename KEEP IT THAT WAY
+# In the event that multiple org members are to be informed of changes
+# to the same file or dir, add them to the end under Multiple Owners
```
Comment (praisenarsie) [inline .github/CODEOWNERS]: fix this file

**Comment by GuyonBroadway** [comment]
Ok the build is about to fial but this time I FUCKING KNOW WHY!!

**Comment by GuyonBroadway** [comment]
ok fingers crossed.

**Comment by JohnGinnane** [comment]
how'd you fix travis

**Comment by GuyonBroadway** [comment]
cyberboss told me a git command to use to update rust-g

**Comment by ghost** [comment]
good shit

**Comment by GuyonBroadway** [comment]
You know I think we can bypass the mapdiffbot if this gets maintainer approval

**Review by praisenarsie — APPROVED** [review]

**Pull request merged**